src/capture/capture.cpp

```diff
@@ -605,7 +605,11 @@ static void capture_init(Section* sec)
 	const std::string prefs = secprop->Get_string("default_image_capture_formats");
 
 	image_capturer = std::make_unique<ImageCapturer>(prefs);
-	video_encoder  = std::make_unique<FfmpegEncoder>();
+	if (secprop->Get_string("video_encoder") == std::string("ffmpeg")) {
+		video_encoder  = std::make_unique<FfmpegEncoder>();
+	} else {
+		video_encoder = std::make_unique<ZMBVEncoder>();
```

Comment:
`ZmbvEncoder` (please don't use all caps for abbreviations, per our convention... there are remnants of that, yes, but eventually those should be fixed up too)

Reply:
fixed